tdrs-frontend/src/components/UploadReport/UploadReport.jsx

```diff
@@ -81,7 +84,7 @@ function UploadReport({ handleCancel, header }) {
       axiosInstance.post(
         // update to `process.env.REACT_APP_BACKEND_URL` and
         // remove mirage route when ready
-        `/mock_api/reports/`,
+        `${process.env.REACT_APP_BACKEND_URL}/reports/`,
```

Comment:
Note that we did need to update this URL here now instead of using Mirage so as to implement this feature.